src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -535,6 +535,7 @@ def load_arguments(self, _):
         c.argument('skip_resource_name_params', action='store_true')
         c.argument('skip_all_params', action='store_true')
         c.argument('resource_ids', nargs='+', options_list='--resource-ids')
+        c.argument('export_format', help="The format of the exported template. Allowed values: 'json', 'arm', 'bicep'. Defaults to 'json'.")
```

Comment:
Could you define an enumeration for the valid values of these parameters?

code example: https://github.com/Azure/azure-cli/blob/5f689fc71cce114699133fdd580843c4132dae1b/src/azure-cli/azure/cli/command_modules/resource/_params.py#L477